app/main.py

```diff
@@ -1,2 +1,53 @@
+from math import ceil
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 0
+        self.capacity = 11
```

Comment:
why self.capacity = 11 if it should be equal to 8 by default ?

Reply:
There is nothing in the README about the required size. I know it`s 8 by default, but I like number 11 better, and as it`s my implementation, why not? I added used ceil for load factor, so 2/3 will be marginally achieved.

Reply:
You should implement a custom dictionary but with the same logic as the built-in python dictionary